spec/features/accessibility/idv_pages_spec.rb

```diff
@@ -32,47 +24,15 @@
       expect_page_to_have_no_accessibility_violations(page)
     end
 
-    scenario 'cancel idv' do
+    scenario 'doc auth steps accessibility' do
       sign_in_and_2fa_user
 
       visit idv_cancel_path
-
       expect(current_path).to eq idv_cancel_path
       expect_page_to_have_no_accessibility_violations(page)
-    end
-
-    scenario 'phone info' do
-      sign_in_and_2fa_user
-      visit idv_path
-      complete_all_doc_auth_steps
-
-      expect(current_path).to eq idv_phone_path
-      expect_page_to_have_no_accessibility_violations(page)
-    end
-
-    scenario 'review page' do
-      sign_in_and_2fa_user
-      visit idv_path
-      complete_all_doc_auth_steps_before_password_step
-
-      expect(page).to have_current_path(idv_enter_password_path)
-      expect_page_to_have_no_accessibility_violations(page)
-    end
 
-    scenario 'personal key / confirmation page' do
-      sign_in_and_2fa_user
       visit idv_path
-      complete_all_doc_auth_steps_before_password_step
-      fill_in t('idv.form.password'), with: Features::SessionHelper::VALID_PASSWORD
-      click_continue
-
-      expect(current_path).to eq idv_personal_key_path
       expect_page_to_have_no_accessibility_violations(page)
-    end
-
-    scenario 'doc auth steps accessibility' do
-      sign_in_and_2fa_user
-      visit idv_path
       complete_all_doc_auth_steps_before_password_step(expect_accessible: true)
       fill_in t('idv.form.password'), with: Features::SessionHelper::VALID_PASSWORD
       click_continue
```

Comment:
Seems like we're missing accessibility checks for a lot of these screens. I'd wonder if we could throw it in here for the password step?

Reply:
Yeah, I think it'd probably be useful to have some expectation of which pages have been checked to make sure we're covering everything we expect to, but I'd leave that to another PR?